.env.example

```diff
@@ -1 +0,0 @@
-REDIS_PASSWORD=[can be anything you want]
```

Comment:
We now have a new .env file at the root of the project, we shouldn't remove the .env.example files since it will be useful for new devs

Reply:
ah ok sorry i will restore it